cloud/README.md

```diff
@@ -0,0 +1,91 @@
+# Cloud Platform Dev Ops
+
+Ansible playbooks to create and maintain cloud based resources.
+
+# Overview
+
+Google Cloud Platform, AWS and Azure all provide platform and infrastructure as a service offerings that are increasingly used by organisations looking to save money and time on configuring infrastructure and application resources. The Ansible scripts included here help with common tasks around cloud provisioning.  
```

Comment:
If so, spelling out an example workflow with how to go from this notebook to provision a new instance, followed by configuring the 'vault.yml' and '.env' of the timemap_datasheet playbook would be helpfully instructive, I think. :)

Reply:
I left it out of Time Map as I wasn't sure if it might be necessary for people working with mtriage to be able to spin machines up and run workload on them. Plus I'm assuming different people will run on different cloud platforms so didn't want to tie GCP in tightly to TimeMap. Totally agree on the workflow and still got some stuff to do around parameterising. Will pull the scripts into TimeMap and update with a sample workflow. GCP and AWS have Platform as a Service offerings that might be worth investigating longer-term but can talk about that another time.